index.js

```diff
@@ -2,32 +2,65 @@ const core = require('@actions/core');
 const exec = require('@actions/exec');
 const github = require('@actions/github');
 
-const ARGS = [
+const ALL_ARGS = [
     'run', '--all-files', '--show-diff-on-failure', '--color=always'
 ];
 
+const CHANGED_ARGS = [
+    'run', '--show-diff-on-failure', '--color=always', '--files'
```

Comment:
`--from-ref` and `--to-ref` are much better than `--files` (the latter can run into argument length limits)

Reply:
I appear to have had an old version installed and didn't have those.  That's definitely better.

Reply:
they used to be called `--origin` / `--source` (and have existed [for quite a while](https://github.com/pre-commit/pre-commit/blob/master/CHANGELOG.md#035---2015-01-15)) (still around as aliases)